nemo/collections/nlp/parts/nlp_overrides.py

```diff
@@ -308,6 +310,111 @@ def save_to(self, model, save_path: str):
         else:
             return super().save_to(model, save_path)
 
+    def restore_from(
```

Comment:
Do we have to duplicate the whole method here? Can we call base class restore_from and then override ? Or extract some additional parameters and use auxiliary method ?